src/frontend/apps/standalone_site/src/features/Contents/routes.tsx

```diff
@@ -0,0 +1,113 @@
+    id: 'routes.routes.menuContentsLivesLabel',
+  },
+  menuContentsClassroomLabel: {
+    defaultMessage: 'Classrooms',
+    description: 'Label for the Classroom link in the content navigation menu',
+    id: 'routes.routes.menuContentsClassroomsLabel',
+  },
+  menuContentsClassroomCreateLabel: {
+    defaultMessage: 'Create Classroom',
+    description: 'Label for the Create Classrooms link',
+    id: 'routes.routes.menuContentsClassroomsCreateLabel',
+  },
+  menuContentsLessonsLabel: {
+    defaultMessage: 'Lessons',
+    description: 'Label for the Lessons link in the content navigation menu',
+    id: 'Contents.routes.menuContentsLessonsLabel',
+  },
+  menuMyContentsLabel: {
+    defaultMessage: 'My Contents',
+    description: 'Label for the MyContents link in the main navigation menu',
+    id: 'Contents.routes.menuMyContentsLabel',
+  },
+});
+
+const routesContent = useContentFeatures.getState().featureRoutes;
+
+const routes: Record<'CONTENTS', RouteRequired> = {
+  CONTENTS: {
+    label: <FormattedMessage {...messages.menuMyContentsLabel} />,
+    path: `/my-contents`,
+    menuIcon: (
+      <LoadSVG
+        Icon={VueListIcon}
+        aria-label="svg-menu-my-contents"
+        title={messages.menuMyContentsLabel}
+      />
+    ),
+    subRoutes: {
+      ...routesContent,
+      LIVE: {
+        label: <FormattedMessage {...messages.menuContentsLivesLabel} />,
+        path: `/my-contents/webinars`,
+        menuIcon: (
+          <LoadSVG
+            Icon={LiveIcon}
+            aria-label="svg-menu-my-contents-live"
+            title={messages.menuContentsLivesLabel}
+          />
+        ),
+        subRoutes: {
+          CREATE: {
+            path: `/my-contents/webinars/create`,
+          },
+          UPDATE: {
+            path: `/my-contents/webinars/:liveId`,
+          },
+        },
+        isNavStrict: true,
+      },
+      CLASSROOM: {
+        label: <FormattedMessage {...messages.menuContentsClassroomLabel} />,
+        path: `/my-contents/classroom`,
+        menuIcon: (
+          <LoadSVG
+            Icon={ClassroomsIcon}
+            aria-label="svg-menu-my-contents-classroom"
+            title={messages.menuContentsClassroomLabel}
+          />
+        ),
+        subRoutes: {
+          CREATE: {
+            path: `/my-contents/classroom/create`,
+          },
+          UPDATE: {
+            path: `/my-contents/classroom/:classroomId`,
+          },
+          INVITE: {
+            path: `/my-contents/classroom/:classroomId/invite/:inviteId`,
+          },
+        },
+        isNavStrict: true,
+      },
+      // LESSON: {
+      //   label: <FormattedMessage {...messages.menuContentsLessonsLabel} />,
+      //   path: `/my-contents/lessons`,
+      //   menuIcon: (
+      //     <CheckListIcon
+      //       width={30}
+      //       height={30}
+      //       role="img"
+      //       aria-label="svg-menu-my-contents-lessons"
+      //     />
+      //   ),
+      // },
+    },
```

Comment:
should be removed ?

Reply:
[🔥(website) remove code about lessons](https://github.com/openfun/marsha/pull/2253/commits/581d5cf80434f0acd45f622d234c5f2fbfa43a34)